js/game_for_mail.js

```diff
@@ -1,7 +1,7 @@
 const email = document.getElementById('email-form');
-const strWindowFeatures = '_blank, height=612, width=820, resizable=no, modal=yes, alwaysRaised=yes, scrollbars=no, menubar=no, location=no, status=no';
+const strWindowFeatures = '_blank, height=910, width=820, resizable=no, modal=yes, alwaysRaised=yes, scrollbars=no, menubar=no, location=no, status=no';
```

Comment:
U mnie jest problem z tą wysokością, bo nie widzę całego layoutu gry, muszę scrollować żeby zobaczyć kotka. Ale to pewnie kwestia wlk ekranu.